src/system-image.js

```diff
@@ -55,7 +55,6 @@ var installLatestVersion = options => {
       .waitForDevice()
       .then(() => adb.wipeCache())
       .then(() => utils.log.debug("adb wiped cache"))
-      .then(() => adb.shell("mount -a"))
```

Comment:
so something like 

```
      .then(() => adb.shell("mount /cache"))
      .then(() => utils.log.debug("adb mounted cache"))
      .then(() => adb.shell("mount /system"))
      .then(() => utils.log.debug("adb mounted system"))
      .then(() => adb.shell("mount /cache"))
```

Reply:
Okay maybe this is safer. Will try both options ;)

Reply:
Yes I had to do it like this, -F is not available on adb shell